### PR TITLE
Questions model

### PR DESCRIPTION
**Pull request by komaltamu**
This branch has the following updates:
- questions database seeding
- model for querying questions
- tests cases for scenarios are not working 

Relevant files to look at:
questions_controller.rb
select_language.html
get_questions.html
routes.rb

### CODE REVIEW

File: features/step_definitions/questions_steps.rb
Comment (sidbav) [inline features/step_definitions/questions_steps.rb:1]: Seems like the merge of two branches did not take place properly! Make sure you test a branch before pushing changes!

File: features/step_definitions/survey_steps.rb [hunk 1]
```diff
@@ -58,11 +64,18 @@
...
+  end
+end
+
+Then(/^I should see the "English", "Spanish", and "Chinese" links$/) do
+  expect(page).to have_link("English")
+  expect(page).to have_link("Spanish")
+  expect(page).to have_link("Chinese")
+end
+
 # #done
```
Comment (sidbav) [inline features/step_definitions/survey_steps.rb:79]: There is quite a bit of commented out code in all of these files -> lets delete whatever we do not need, if we may potentially need something in the future add a comment explaining why we are keeping that code!

**Review by sidbav — CHANGES_REQUESTED** [review]
Currently does not work! See comments below!

File: features/step_definitions/questions_steps.rb
Comment (chinmay10) [reply features/step_definitions/questions_steps.rb:1]: We have worked on the testcases now they are working, I and komal will have to sort out the commit issues and then we will add a new pull request. 

Initially thats why we were reluctant to create PR for this.

Comment (sidbav) [reply features/step_definitions/questions_steps.rb:1]: Ah i see, if thats the case, feel free to close this PR. Open new one once changes are ready!!

**Pull request closed without merging**